assets/users/tilde.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Want to run this through `svgo`? Just `yarn global add svgo` (or `npm install --global svgo`) and `svgo tilde.svg`